src/bin/main.rs

```diff
@@ -30,12 +32,33 @@ async fn main() -> std::io::Result<()> {
     let auth = Arc::new(AuthorizationService::new(cfg.clone(), database.clone()));
     let tracker_service = Arc::new(TrackerService::new(cfg.clone(), database.clone()));
     let mailer_service = Arc::new(MailerService::new(cfg.clone()).await);
+
+    let max_image_request_timeout_ms = cfg.settings.read().await.cache.image_cache_max_request_timeout_ms;
+    let max_image_size = cfg.settings.read().await.cache.image_cache_entry_size_limit;
+    let user_quota_period_seconds = cfg.settings.read().await.cache.image_cache_user_quota_period_seconds;
+    let user_quota_bytes = cfg.settings.read().await.cache.image_cache_user_quota_bytes;
+    let image_cache_capacity = cfg.settings.read().await.cache.image_cache_capacity;
+
+    let image_cache_manager_config = ImageCacheManagerConfig {
+        max_image_request_timeout_ms,
+        max_image_size,
+        user_quota_period_seconds,
+        user_quota_bytes,
+    };
```

Comment:
Hi @josecelano , I see the image cache as an isolated plugin to the backend. This is why I thought that it should own its own config struct.

Reply:
OK, you mean it could have its crate without specifying how the configuration should be stored in a toml file, do you? I suppose that kind of extension should also provide a specification for a config file section. But maybe they could also use an independent config file. Anyway, for the time being, `config.settings.get_cache_configuration` could return an `ImageCacheManagerConfig`. I think that's easy to change in the future if you extract the extension.